locations/spiders/enjoy_tacos.py

```
@@ -0,0 +1,21 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import Categories, apply_category
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+
+
+class EnjoyTacosSpider(SitemapSpider, StructuredDataSpider):
+    name = "enjoy_tacos"
+    item_attributes = {"brand": "Enjoy Tacos", "brand_wikidata": "Q141242529"}
+    sitemap_urls = ["https://enjoytacos.belorder.com/sitemap.xml"]
+    sitemap_rules = [(r"https\:\/\/enjoytacos\.belorder\.com\/\/restaurants\/.+$", "parse_sd")]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)